datarepo-clienttests/src/main/java/common/utils/KubernetesClientUtils.java

```diff
@@ -370,6 +371,7 @@ private static long getApiReadyPods(V1Deployment deployment) throws ApiException
                 pod ->
                     deploymentComponentLabel.equals(
                             pod.getMetadata().getLabels().get(componentLabel))
```

Comment:
Since `componentLabel` is a constant it should be written as `COMPONENT_LABEL`.

Reply:
That would make it inconsistent with how constants are defined throughout the datarepo-clienttests project. I'm okay with changing the standard, but I think we should do it everywhere at once and in a separate PR. It looks like there some inconsistency in the main data repo project also (DatasetDao, DatasetTableDao), so would be good to clean it all up at once.